jobsdb/jobsdb.go

```diff
@@ -97,24 +97,27 @@ HandleT is the main type implementing the database for implementing
-	tableFileDumpTimeStat, fileUploadTimeStat, totalTableDumpTimeStat, jobsdbQueryTimeStat *stats.RudderStats
+	dbHandle                                                                                                                                                                       *sql.DB
+	tablePrefix                                                                                                                                                                    string
+	datasetList                                                                                                                                                                    []dataSetT
+	datasetRangeList                                                                                                                                                               []dataSetRangeT
+	dsListLock                                                                                                                                                                     sync.RWMutex
+	dsMigrationLock                                                                                                                                                                sync.RWMutex
+	dsRetentionPeriod                                                                                                                                                              time.Duration
+	dsEmptyResultCache                                                                                                                                                             map[dataSetT]map[string]map[string]map[string]bool
+	dsCacheLock                                                                                                                                                                    sync.Mutex
+	backupMux                                                                                                                                                                      *sync.Mutex
+	toBackup                                                                                                                                                                       bool
+	backupOnlyAborted                                                                                                                                                              bool
+	jobsFileUploader                                                                                                                                                               filemanager.FileManager
+	jobStatusFileUploader                                                                                                                                                          filemanager.FileManager
+	jobStatusAbortedFileUploader                                                                                                                                                   filemanager.FileManager
```

Comment:
Instead of a separate uploader, let's drive it through the configuration. (Eg: jobsdb.gw.backupOnlyAborted => should only backup aborted rows)

Reply:
changed to one uploader